client/update.go

```diff
@@ -411,3 +419,8 @@ func makeChannelUpdate(next *channel.State, actor channel.Index) ChannelUpdate {
 		ActorIdx: actor,
 	}
 }
+
```

Comment:
I still think it is sad that we now mix error checking functions like `Is…Error` and `errors.Is` but OK… its too late now.

Reply:
I have used `errors.As` in the tests for the newly added error. Should I instead add an `Is..Error` function that internal uses `errors.Cause` as in other places in the project ? @ggwpez

Reply:
It would be nice if it was consistent across the framework. On the other hand, I don't like introducing Is...Error functions all over the place, as this is pretty much redundant to the errors.Is functionality. Or am I wrong on this? Any other views and suggestions on this are appreciated.

Reply:
I agree with you @matthiasgeihs. Personally I also prefer to use `errors.Is` over `Is..Error` function.

Reply:
Ok then lets leave it as is.